client/my-sites/people/invite-people/index.jsx

```diff
@@ -181,19 +174,29 @@ class InvitePeople extends React.Component {
 	onRoleChange = ( event ) => {
 		const role = event.target.value;
 		this.setState( { role } );
-		createInviteValidation( this.props.siteId, this.state.usernamesOrEmails, role );
+		this.validateInvitation( this.props.siteId, this.state.usernamesOrEmails, role );
 	};
 
 	onExternalChange = ( event ) => {
 		const isExternal = event.target.checked;
 		this.setState( { isExternal } );
 	};
 
-	refreshValidation = () => {
-		const errors =
-			InvitesCreateValidationStore.getErrors( this.props.siteId, this.state.role ) || {};
-		const success =
-			InvitesCreateValidationStore.getSuccess( this.props.siteId, this.state.role ) || [];
+	async validateInvitation( siteId, usernamesOrEmails, role ) {
```

Comment:
Note: I (again) chose the approach using internal methods here because it's much simpler and the API response isn't used anywhere else.